apiserver/params/crossmodel.go

```diff
@@ -146,7 +146,7 @@ type OfferURLs struct {
 // ConsumeApplicationArg holds the arguments for consuming a remote application.
 type ConsumeApplicationArg struct {
 	// The offer to be consumed.
-	ApplicationOffer
+	ApplicationOfferDetails
```

Comment:
ISTM what you really want is:

```go
type ApplicationOffer struct {
    ApplicationOfferIdentity
    ApplicationOfferDetails
}
```

Then you would use ApplicationOfferIdentity here. The rest of the details aren't relevant here, are they?

Reply:
As discussed, all this info is used.